### ISSUE TITLE
pull-kubernetes-verify are failing in 1.13

### ISSUE THREAD

**Issue by jfbai** [kubernetes/kubernetes#80557]
**Which jobs are failing**:

pull-kubernetes-verify

**Which test(s) are failing**:

`verify.godeps`
https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/80468/pull-kubernetes-verify/1154022534891442176/
https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/80468/pull-kubernetes-verify/1153713369983750144/

**Since when has it been failing**:

Jul 24, 2019, 1:08 AM GMT +8

**Testgrid link**:

https://testgrid.k8s.io/sig-release-1.13-blocking#verify-1.13

**Reason for failure**:

```
 		{
 			"ImportPath": "github.com/libopenstorage/openstorage/api",
+			"Comment": "v1.0.0",
 			"Rev": "093a0c3888753c2056e7373183693d670c6bba01"
 		},
 		{
 			"ImportPath": "github.com/libopenstorage/openstorage/api/client",
+			"Comment": "v1.0.0",
 			"Rev": "093a0c3888753c2056e7373183693d670c6bba01"
 		},
 		{
 			"ImportPath": "github.com/libopenstorage/openstorage/api/client/volume",
+			"Comment": "v1.0.0",
 			"Rev": "093a0c3888753c2056e7373183693d670c6bba01"
 		},
 		{
 			"ImportPath": "github.com/libopenstorage/openstorage/api/spec",
+			"Comment": "v1.0.0",
 			"Rev": "093a0c3888753c2056e7373183693d670c6bba01"
 		},
 		{
 			"ImportPath": "github.com/libopenstorage/openstorage/pkg/parser",
+			"Comment": "v1.0.0",
 			"Rev": "093a0c3888753c2056e7373183693d670c6bba01"
 		},
 		{
 			"ImportPath": "github.com/libopenstorage/openstorage/pkg/units",
+			"Comment": "v1.0.0",
 			"Rev": "093a0c3888753c2056e7373183693d670c6bba01"
 		},
 		{
 			"ImportPath": "github.com/libopenstorage/openstorage/volume",
+			"Comment": "v1.0.0",
 			"Rev": "093a0c3888753c2056e7373183693d670c6bba01"
 		},
 		{
@@ -2983,7 +2990,7 @@
 		},
 		{
 			"ImportPath": "github.com/robfig/cron",
-			"Comment": "v1-53-gdf38d32658d878",
+			"Comment": "v1.0.0-53-gdf38d32658d878",
 			"Rev": "df38d32658d8788cd446ba74db4bb5375c4b0cb3"
 		}
```

**Anything else we need to know**:

None

**Comment by spiffxp** [comment]
What ended up being the problem / resolution here?

**Comment by jfbai** [comment]
> What ended up being the problem / resolution here?

Fixed by #80559
